recipe/meta.yaml

```diff
@@ -28,13 +29,19 @@ requirements:
 test:
   imports:
     - aiohttp_jinja2
+  requires:
+    - pip
+    - python <3.10
+  commands:
+    - pip check
 
 about:
   home: https://github.com/aio-libs/aiohttp-jinja2/
   license: Apache Software
```

Comment:
License should be `Apache-2.0`

Reply:
updated the license:

https://github.com/AnacondaRecipes/aiohttp-jinja2-feedstock/blob/update_to_1.5/recipe/meta.yaml#L41